lib/utils.js

```diff
@@ -496,3 +496,20 @@ class Where extends SequelizeMethod {
 exports.Where = Where;
 
 exports.validateParameter = parameterValidator;
+
+
+exports.mapIsolationLevelStringToTedious = isolationLevel => {
+  const tediousIsolationLevel = require('tedious').ISOLATION_LEVEL;
```

Comment:
expect `lib` in second argument, use that `tedious` instance if available, otherwise default to `require('tedious')`.

Or may be we should just enforce passing lib instance in second argument or this method should throw, wdyt ? @harshithkashyap